dll/win32/shlwapi/shlwapi.spec

```diff
@@ -636,7 +637,8 @@
 636 stdcall PathIsSameRootA(str str)
 637 stdcall PathIsSameRootW(wstr wstr)
 638 stdcall PathIsSystemFolderA(str long)
-639 stdcall PathIsSystemFolderW(wstr long)
+639 stdcall -noname -version=0x600+ SHInvokeCommandOnContextMenuEx(ptr ptr ptr long long str wstr)
+@ stdcall PathIsSystemFolderW(wstr long)
```

Comment:
Same here, what happens on `0x600+`. Should it be replaced by `SHInvokeCommandOnContextMenuEx` maybe?
In that case, there should be a max version on this.